pandas/core/window.py

```diff
@@ -850,6 +850,8 @@ def validate(self):
         super(Rolling, self).validate()
         if not com.is_integer(self.window):
             raise ValueError("window must be an integer")
+        elif self.window < 0:
```

Comment:
there is a another place where this is checked. in _Window (so need a test for that as well)